src/main/java/org/opentripplanner/routing/edgetype/ParkAndRideEdge.java

```diff
@@ -51,7 +51,6 @@ public State traverse(State s0) {
             s1.incrementWeight(time);
             s1.incrementTimeInSeconds(time);
             s1.setCarParked(false);
-            s1.setBackMode(TraverseMode.LEG_SWITCH);
```

Comment:
We will want to test that paths containing a park and ride transition are still properly broken into legs after these changes. It seems like they would be, but probably worth testing.

Reply:
I did some tests of the park and ride functionality using the built-in client, and I got the the proper car->walk->transit transitions.